src/gateway/how-kong-works/routing-traffic.md

```diff
@@ -588,10 +539,8 @@ defined URIs, in this order:
 3. `/version/any/`
 4. `/version`
 
-Take care to avoid writing Regex rules that are overly broad and may consume
-traffic intended for a prefix rule. Adding a rule with the path `/version/.*` to
-the ruleset above would likely consume some traffic intended for the `/version`
-prefix path. If you see unexpected behavior, sending `Kong-Debug: 1` in your
+Routers that employ a large amount of regexes can consume traffic intended for other rules.
```

Comment:
```suggestion
Routers with a large number of regexes can consume traffic intended for other rules.
```
Is this a good thing or bad thing? It almost feels like it's missing a "This results in...."

Reply:
bad thing, should I make it more clear?